### ISSUE TITLE
constraint::stock::params::tests::test_handle_call_fail fails on stable Rust

### ISSUE THREAD

**Issue by asomers** [pcsm/simulacrum#50]
constraint::stock::params::tests::test_handle_call_fail fails on Rust 1.25.0 stable and 1.27.0 beta.  It passes on 1.27.0 nightly.  Apart from simulacrum_auto, this is the only thing that fails on stable Rust.
```
---- constraint::stock::params::tests::test_handle_call_fail stdout ----
        thread 'constraint::stock::params::tests::test_handle_call_fail' panicked at 'assertion failed: `(left == right)`
  left: `MismatchedParams("<Always Fails>", "<unknown>")`,
 right: `MismatchedParams("<Always Fails>", "()")`: Constraint should return the correct error', simulacrum_mock/src/constraint/stock/params.rs:86:9
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by jasongrlicky** [comment]
Thanks for reporting this! I'll look into it - it seems like it's a case of an overly-specific test.

**Comment by jasongrlicky** [comment]
Fixed in 6b5869417573fff5d88a5cd188f08fb14b524886 - thanks again for reporting!
